templates/azure/fips.html

```diff
@@ -347,7 +395,7 @@ <h2>Pricing</h2>
           </tr>
         </tbody>
       </table>
-      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips">Ubuntu Pro FIPS 18.04 LTS on the Azure Marketplace</a></p>
+      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips">Ubuntu Pro FIPS 20.04 LTS on the Azure Marketplace</a></p>
```

Comment:
`>` is missing here and in line 318
```suggestion
      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips">Ubuntu Pro FIPS 20.04 LTS on the Azure Marketplace&nbsp;&rsaquo;</a></p>
```